### PR TITLE
Docker 1.8.2's initial startup is really slow

### PR DESCRIPTION
**Pull request by brenton**

### CODE REVIEW

**Comment by brenton** [comment]
@danmcp, PTAL

I'll make sure someone is aware of this problem.  For now this unbreaks the base ami job for rhel7next.  I'm not sure how it worked for me last week honestly.

File: lib/vagrant-openshift/action/install_openshift_base_dependencies.rb [hunk 1]
```diff
@@ -121,6 +121,10 @@ def call(env)
 
 sed -i "s,^#DefaultTimeoutStartSec=.*,DefaultTimeoutStartSec=240s," /etc/systemd/system.conf
 
+# Docker 1.8.2 now sets a TimeoutStartSec of 1 minute.  Unfortunately, for some
+# reason the initial docker start up is now taking > 5 minutes.  Until that is fixed need this.
+sed -i 's,TimeoutStartSec=.*,TimeoutStartSec=10min,'  /usr/lib/systemd/system/docker.service
+
```
Comment (danmcp) [inline lib/vagrant-openshift/action/install_openshift_base_dependencies.rb]: The line above is for the same thing.  We probably only need this one.

Comment (brenton) [inline lib/vagrant-openshift/action/install_openshift_base_dependencies.rb]: Well, I didn't want to change the behavior and my hope was to nuke this asap.  Technically my addition has no affect to the unit file for docker < 1.8.

Comment (brenton) [inline lib/vagrant-openshift/action/install_openshift_base_dependencies.rb]: If you prefer I'll just explicitly update /usr/lib/systemd/system/docker.service for all docker versions and remove the previous workaround.

Comment (danmcp) [inline lib/vagrant-openshift/action/install_openshift_base_dependencies.rb]: No that's fine. Thanks for the explanation.

**Pull request merged**